game.cpp

```diff
@@ -1902,6 +1902,33 @@ void test_game() //!OCLINT tests may be many
   }
   #endif // FIX_ISSUE_241
 
+//#define FIX_ISSUE_457
+#ifdef FIX_ISSUE_457
+  {
+    // (457) The color of any player can be accessed easily
+    game g;
+    color color_player_one = get_nth_player_color(g, 0);
+    color color_player_two = get_nth_player_color(g, 1);
+    color color_player_three = get_nth_player_color(g, 2);
+    // Default colors for players: R, G, B
+    assert(color_player_one == create_red_color());
+    assert(color_player_two == create_green_color());
+    assert(color_player_three == create_blue_color());
+  }
+#endif
+
+//#define FIX_ISSUE_458
+#ifdef FIX_ISSUE_458
+  {
+    // (458) The color of any food item can be accessed easily
+    game g;
+    color color_food = get_nth_food_color(g, 0);
+    // Default color for the food item is black
+    color c;
+    assert(color_food == c);
```

Comment:
This is scary, as the comment is needed to explain the code, where these may very likely change in the future.

```c++
assert(color_food == color::black);
assert(color_food == create_black());
assert(color_food == create_black_color());
```

Reply:
The test now expects the food color to be the default color (of the class). No need for a possibly lying comment anymore!